sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -313,11 +313,73 @@ for (int i = 0; i < receiptPageResults.size(); i++) {
 }
 ```
 
+### Recognize business cards
+Recognize data from business cards using a prebuilt model. Business card fields recognized by the service
+can be found [here][service_recognize_business_card].
+See [StronglyTypedRecognizedForm][strongly_typed_sample] for a suggested approach to extract
+information from a business card. The given sample is for `receipt` but the same approach applies to `business card`.
```

Comment:
I would recommend adding a simialr sample for businesscards.

Reply:
as discussed in the scrum. Won't need to add a stronglyTyped sample for Buisness Card